policyengine_us/variables/gov/states/mt/tax/income/deductions/cdcc/mt_child_dependent_care_expense_deduction.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class mt_child_dependent_care_expense_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana child dependent care expense deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
```

Comment:
reorganize this code to align with the sequence in the form: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2022/12/2441-M_2022.pdf

and break it up more for clarity, e.g. `reduction = p.reduction.calc(agi)`

Reply:
this still doesn't align with the sequence in the form

Reply:
aligned and added line numbers for clarity